cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/realtime/kafka/Kafka08SimpleApiConsumer.java

```diff
@@ -69,16 +69,18 @@
 import javax.annotation.Nullable;
 
 /**
- * <p>
- *  The {@link KafkaSimpleApiConsumer} compatible mode for {@code Kafka} 0.8.x releases.
+ * The {@link KafkaSimpleApiConsumer} compatible mode for {@code Kafka} 0.8.x releases.
```

Comment:
I thought when you have a ``<p>``, you indent the next line. But I will take your word for this :)